packages/graph-editor/src/hooks/useClickOutside.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useRef } from 'react';
+
+export default <T extends HTMLElement>(
+  callback: () => void,
+  active = false,
+) => {
+  const ref = useRef<T>(null);
+
+  useEffect(() => {
+    const handleClick = (event: MouseEvent) => {
+      const target = event.target as HTMLElement;
+      //TODO: find a solution for this
+      if (
+        ref.current &&
+        !ref.current.contains(target) &&
+        target.nodeName !== 'BODY' &&
+        target.nodeName !== 'HTML'
+      ) {
+        if (
+          [...target.classList.values()].find((itm) =>
```

Comment:
we probably do not want this here. These are some workarounds for studio. So maybe remove and test if it works without